### ISSUE TITLE
Open Source Guides: Places to congregate

### ISSUE THREAD

**Issue by theo-armour** [github/opensource.guide#384]
Open Source Guides identifies a number of topics that when followed set a high and welcoming bar for open source projects.

As people begin to follow these guides they may want to share their successes, comments and ideas with similarly minded peeps.

In order to help with this sharing, would it be a good thing to establish the handles for sharing? For example,:
* 'open-source-guides' could be a suitable GitHub Topic to add to repositories 
* '#opensourceguides.' could be a Twitter hashtag 

This information could be added to the Guides' README file.

It might also be a good thing to add additional relevant GitHub Topics to the Guides repository - again as examples to be followed.

**Comment by kytrinyx** [comment]
Thank you @theo-armour.

> As people begin to follow these guides they may want to share their successes, comments and ideas with similarly minded peeps.

It would be great to hear people's stories, and it does seem like this would (naturally) happen elsewhere, since it does not make sense to open a GitHub issue for something like that.

Have you seen people sharing their experiences anywhere? I have not (yet) seen anything, and it seems a bit premature to try to regulate behavior that hasn't happened yet.

I did a search on Twitter, and at the moment there are a small handful of tweets that contain `#opensourceguide` and even fewer that came up under `#opensourceguides`. None of the tweets that I found were about people's experiences, only people sharing the guides themselves.

I'd suggest that we hold off on explicitly suggesting a way for people to share their stories, until we get an idea of how people are actually using the guides and where/how they share their experiences.

> It might also be a good thing to add additional relevant GitHub Topics to the Guides repository - again as examples to be followed.

We had 'open-source' as a topic. I've added 'oss' and 'best-practices'. I would suggest that if we want to discuss suitable topics for this repository, that we do so in a separate issue, and keep this focused on the "sharing experiences" question.
